### DISCUSSION TITLE
Wildcards manager is great!

### DISCUSSION THREAD

**Discussion by rpillala** [adieyal/sd-dynamic-prompts#112] [General]
I love the wildcards manager tab thanks for thinking about this quality of life improvement!

**Comment by adieyal** [comment]
More to come... :)

**Comment by adieyal** [comment]
Could you please mark this question as answered? Otherwise it is hard for me to keep track of unanswered questions.

**Reply by rpillala** [reply]
I don't see a way to mark this one as answered.  I didn't create it as a question type.  I don't see a way to delete it either.

**Reply by adieyal** [reply]
no worries
